students/10th/soojungsohn/user/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+# Create your models here.
+class Users(models.Model):
```

Comment:
클래스 이름은 단수로 사용합니다 :) 
```suggestion
class User(models.Model):
```